modules/cli/src/main/scala/scala/cli/commands/pgp/PgpScalaSigningOptions.scala

```diff
@@ -0,0 +1,30 @@
+package scala.cli.commands.pgp
+
+import caseapp.*
+
+import scala.build.options as bo
+import scala.cli.commands.shared.{CoursierOptions, LoggingOptions, SharedJvmOptions}
+
+// format: off
+final case class PgpScalaSigningOptions(
+  @Hidden
+    signingCliVersion: Option[String] = None,
+  @ValueDescription("option")
+  @Hidden
+    signingCliJavaArg: List[String] = Nil,
+  @HelpMessage("Whether to run the Scala Signing CLI on the JVM or using a native executable")
+  @Hidden
+    signingCliOnJvm: Option[Boolean] = None
```

Comment:
NIT Mixed feelings on the current name for this flag.
How about:
```suggestion
  @HelpMessage("Specify the platform to run Scala Signing CLI (defaults to running the native executable)")
  @ValueDescription("native|jvm")
  @Hidden
    signingCliPlatform: Option[String] = None
```
or if we want to keep it as a boolean flag:
```suggestion
  @HelpMessage("Forces the JVM platform for Scala Signing CLI")
  @Hidden
  @Name("forceJvmSigning")
    forceJvmSigningCli: Option[Boolean] = None
```

Reply:
Right, `forceJvmSigningCli` it much better than `signingCliOnJvm`.  But I wouldn't add any alias, this flag is added only for test purpose.